ci-operator/step-registry/openshift-extended/test/disasterrecovery/sanity/openshift-extended-test-disasterrecovery-sanity-chain.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "openshift-extended/test/disasterrecovery/sanity/openshift-extended-test-disasterrecovery-sanity-chain.yaml",
```

Comment:
@wangke19 I just find you put chain in wrong place. this directory is only for step, not chain.

for chain, you should put it in the directory same to chain @geliu2016 ever create DR chain.

and no need to add xzha and me into approver and reviewer.